middleware/jwt/src/middleware.rs

```diff
@@ -145,10 +147,14 @@ where
         if token.is_none() {
             trace!("[{}] bad request jwt middleware", request_id(&state));
             let res = create_empty_response(&state, StatusCode::BAD_REQUEST);
-            return Box::new(future::ok((state, res)));
+            return future::ok((state, res)).boxed();
         }
 
-        match decode::<T>(&token.unwrap(), self.secret.as_ref(), &self.validation) {
+        match decode::<T>(
+            &token.unwrap(),
+            &DecodingKey::from_secret(self.secret.as_ref()),
```

Comment:
I think this line would be better outside and stored as a local

Reply:
Yep, done.